### ISSUE TITLE
[Request] Horizontally and vertically align icon with base layer after converting

### ISSUE THREAD

**Issue by joshmtucker** [keremciu/sketch-iconfont#100]
Good morning!

It would be super awesome (and save a ton of time) if the vector icon and the base layer could be automatically aligned (vertically and horizontally) after converting the icon from the font.

Is this possible,

Thanks so much, keep up your great work!

**Comment by keremciu** [comment]
@joshmtucker hello, morning too

can you send me a little screencast video for this? how are you handling this problem? I can find a nice way to find a nice solution by watching it

Cheers
Kerem

**Comment by Raghav789** [comment]
@joshmtucker automatic align isn't the solution here, sometimes google intentionally won't align stuff for example play button the actual problem is upon conversion please refer my issue here #121 and the similar issue here #148 I think we must be missing something huge it seems that only we are facing this issue

**Comment by keremciu** [comment]
@Raghav789 yes, I got the point now. is it only related with material icons, right?

**Comment by Raghav789** [comment]
@keremciu thank you for considering it, I just tried with all icon family and this happens with all icons
